apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V26__Add_fields_to_BeneficiariesInpatient.sql

```diff
@@ -0,0 +1,30 @@
+/*
+ * The column doesn't have a default value to avoid updating the column on migration. The pipeline server
+ * will populate the column as new beneficaries are added or existing beneficaries are updated. 
+ */
+
+alter table "Beneficiaries" add column "derivedMailingAddress1" varchar(40);
+
+alter table "Beneficiaries" add column "derivedMailingAddress2" varchar(40);
+
+alter table "Beneficiaries" add column "derivedMailingAddress3" varchar(40);
+
+alter table "Beneficiaries" add column "derivedMailingAddress4" varchar(40);
+
+alter table "Beneficiaries" add column "derivedMailingAddress5" varchar(40);
+
+alter table "Beneficiaries" add column "derivedMailingAddress6" varchar(40);
+
+alter table "Beneficiaries" add column "derivedCityName" varchar(100);
+
+alter table "Beneficiaries" add column "derivedStateCode" varchar(2);
+
+alter table "Beneficiaries" add column "derivedZipCode" varchar(9);
```

Comment:
How does the data in this field differ from `"Beneficiaries"."postalCode"`?

Reply:
From GEDIT->. The following variables in the current FHIR extract are from the EDB source:

·        STATE_CD

·        CNTY_CD

·        BENE_ZIP

 

These new variables will be from the CME Derived Mailing source:

·        DRVD_LINE_1_ADR

·        DRVD_LINE_2_ADR

·        DRVD_LINE_3_ADR

·        DRVD_LINE_4_ADR

·        DRVD_LINE_5_ADR

·        DRVD_LINE_6_ADR

·        CITY_NAME

 

We are finding some differences in the State Code between the two sources.


So we did find differences which is why we proposed this.